sdk/core/System.Net.ClientModel/tests/client/OpenAIClientTests.cs

```diff
@@ -31,4 +33,16 @@ public void TestClientSync()
 
         Assert.IsTrue(choice.Text.StartsWith("\n\nLife is"));
     }
+
+    [Test]
```

Comment:
For your context, I don't think we are blocking CIs for tests run in this project.  If we want to, we can change this.  For unit tests of types in System.Net.ClientModel, the intention is that those will go in the `System.Net.ClientModel.Tests` project if they are testing public APIs and in the `System.Net.ClientModel.Tests.Internal` project if they are testing internal APIs and need InternalsVisibleTo.

This is a new test structure, but my hypothesis is that it is better suited to the needs of Core libraries that might contain dynamic types and may need to validate nullability contracts in our APIs.  If we decide we like it in ClientModel, we can consider moving Azure.Core test projects to reflect the same structure for consistency.  That said, if it feels too high-friction, I'm open to that feedback as well as we consider what we want for the ClientModel project.